lib/fog/compute/openstack.rb

```diff
@@ -1,8 +1,9 @@
-
-
 module Fog
   module Compute
     class OpenStack < Fog::Service
+      SUPPORTED_VERSIONS = /v2\.0|v2\.1/
+      SUPPORTED_MICROVERSION = '2.15'.freeze
```

Comment:
Why only `2.15`? Not to trigger more questions about how microversions are handled.

Reply:
We're just introducing 2.15 here, because the feature we're adding is available with this microversion. There are two options, I guess: Updating to the currently most recent microversion for nova, or leaving it at 2.15. I'm tending more towards the latter, as I can't verify the changes e.g. against Pike easily. WDYT?

Reply:
Well the issue here is the general handling of microversions (let me save my rant about how microversion is not helping SDKs but serve only Openstack APIs themselves!). Well I don't want to block this PR meanwhile I'm not sure this is the right thing to do, and again that's likely completely inherent to the way microversion works (or how it wants us to work with). So unless someone has a better idea, then we can merge and continue to be doomed with microversion.

Reply:
Sorry, I didn't want to trigger bad feelings about microversion handling on the OpenStack side here :(
I'm entirely with you that it is actually pretty cumbersome to deal with this concept on the SDK or CLI level – seems like the easier way out for the API developers.
So, for the sake of this PR, I'd say we stick to microversion 2.15. I assume that the next person who needs a feature which is supported in >2.15 will have to do a similar thing. Looking at https://docs.openstack.org/nova/latest/reference/api-microversion-history.html I feel like depending on the needed microversion, a lot of non-related changes might to be done, though.

Reply:
@voelzmo,

It seems like we don't have another solution for now and not block this PR.
Looks like we effectively need bumping the micro-version too as the latest is 2.56 but that's a different PR for sure! 
Thanks for joining the conversation, we likely need to continue this on a dedicated issue thread.

Reply:
I've opened https://github.com/fog/fog-openstack/issues/345 to track some of the issues, so the discussion can continue. Thanks for merging!